ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/ViewBasedReportJobDetails.tsx

```diff
@@ -108,17 +108,6 @@ function ViewBasedReportJobDetails({ reportSnapshot }: ViewBasedReportJobDetails
                     <DescriptionListTerm>CVEs discovered since</DescriptionListTerm>
                     <DescriptionListDescription>All time</DescriptionListDescription>
                 </DescriptionListGroup>
-                <DescriptionListGroup>
-                    <DescriptionListTerm>Optional columns</DescriptionListTerm>
-                    <DescriptionListDescription>
-                        <Stack>
-                            {reportSnapshot.vulnReportFilters.includeNvdCvss && <div>NVD CVSS</div>}
-                            {reportSnapshot.vulnReportFilters.includeEpssProbability && (
-                                <div>EPSS probability</div>
-                            )}
-                        </Stack>
-                    </DescriptionListDescription>
-                </DescriptionListGroup>
```

Comment:
Optional columns isn't needed since we're showing all columns in the CSV anyways